content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -185,18 +185,18 @@ Once Pod is assigned to a node by scheduler, kubelet starts creating containers
        Reason:       ErrImagePull
 	  ...
    ```
-   
```

Comment:
@furukawa3 , Line 186, are there two tabs that could be removed and replaced with spaces?

Reply:
ditto https://github.com/kubernetes/website/pull/16787#discussion_r339849626
I'd like to replace them :)